[2024-06-30 07:53:52,265] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_zero 2024-06-30T07:53:42.745664+00:00 [None]>
[2024-06-30 07:53:52,272] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_zero 2024-06-30T07:53:42.745664+00:00 [None]>
[2024-06-30 07:53:52,272] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 07:53:52,273] {taskinstance.py:1088} INFO - Starting attempt 1 of 3
[2024-06-30 07:53:52,273] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 07:53:52,286] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_zero> on 2024-06-30T07:53:42.745664+00:00
[2024-06-30 07:53:52,292] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'assignment', 'task_zero', '2024-06-30T07:53:42.745664+00:00', '--job-id', '3', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmpoj_xgfps', '--error-file', '/tmp/tmp7i31en2u']
[2024-06-30 07:53:52,289] {standard_task_runner.py:52} INFO - Started process 211 to run task
[2024-06-30 07:53:52,292] {standard_task_runner.py:77} INFO - Job 3: Subtask task_zero
[2024-06-30 07:53:52,326] {logging_mixin.py:104} INFO - Running <TaskInstance: assignment.task_zero 2024-06-30T07:53:42.745664+00:00 [running]> on host 2532fdaf14cb
[2024-06-30 07:53:52,368] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=assignment
AIRFLOW_CTX_TASK_ID=task_zero
AIRFLOW_CTX_EXECUTION_DATE=2024-06-30T07:53:42.745664+00:00
[2024-06-30 07:53:52,369] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2024-06-30 07:53:52,370] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'pip install beautifulsoup4']
[2024-06-30 07:53:52,376] {subprocess.py:75} INFO - Output:
[2024-06-30 07:53:53,630] {subprocess.py:79} INFO - Defaulting to user installation because normal site-packages is not writeable
[2024-06-30 07:53:53,894] {subprocess.py:79} INFO - Collecting beautifulsoup4
[2024-06-30 07:53:54,010] {subprocess.py:79} INFO -   Downloading beautifulsoup4-4.12.3-py3-none-any.whl (147 kB)
[2024-06-30 07:53:54,182] {subprocess.py:79} INFO - Collecting soupsieve>1.2
[2024-06-30 07:53:54,195] {subprocess.py:79} INFO -   Downloading soupsieve-2.3.2.post1-py3-none-any.whl (37 kB)
[2024-06-30 07:53:57,719] {subprocess.py:79} INFO - Installing collected packages: soupsieve, beautifulsoup4
[2024-06-30 07:53:57,879] {subprocess.py:79} INFO - Successfully installed beautifulsoup4-4.12.3 soupsieve-2.3.2.post1
[2024-06-30 07:53:58,158] {subprocess.py:83} INFO - Command exited with return code 0
[2024-06-30 07:53:58,181] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=assignment, task_id=task_zero, execution_date=20240630T075342, start_date=20240630T075352, end_date=20240630T075358
[2024-06-30 07:53:58,230] {local_task_job.py:151} INFO - Task exited with return code 1
